PyDSS/utils/utils.py

```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_redirect(file_name):
```

Comment:
Would you mind making a proper docstring?  You can use the other functions in this file as examples.  For the exception you would add something like this:

```
"""
Raises
-------
Exception
    Raised if the command fails.

"""
```

Reply:
Okay, I'll add that